src/Common/src/Interop/Unix/System.Native/Interop.GetSystemTimeAsTicks.cs

```diff
@@ -10,7 +10,7 @@ internal static partial class Interop
 {
     internal unsafe partial class Sys
     {
-        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_GetSystemTimeAsTicks")]
+        [DllImport(Interop.Libraries.SystemNative, EntryPoint = "SystemNative_GetSystemTimeAsTicks")]
```

Comment:
Could you please delete CoreLibNative_GetSystemTimeAsTicks as part of this? (https://github.com/dotnet/corert/blob/master/src/Native/System.Private.CoreLib.Native/pal_datetime.cpp#L16)

Also, I think you will to upgrade the corefx nuget package with System.Native to make this work. It may be best to do this whole operation as separate change.